<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorView</class>
 <widget class="QWidget" name="EditorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QLabel" name="delegateLabel">
     <property name="text">
      <string>Delegate To: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QWidget" name="taskGroup" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="delegateHBox" stretch="0,0">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="delegateToLabel">
          <property name="text">
           <string>De&amp;legate to</string>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="delegateEditPlaceHolder" native="true">
          <layout class="QHBoxLayout" name="delegateEditPlaceHolderLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="startDateHBox" stretch="0,1,0,1">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="startDateLabel">
          <property name="text">
           <string>St&amp;art Date</string>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="buddy">
           <cstring>startDateEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPIM::KDateEdit" name="startDateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="dueDateLabel">
          <property name="text">
           <string>D&amp;ue Date </string>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <property name="buddy">
           <cstring>dueDateEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPIM::KDateEdit" name="dueDateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="startTodayButton">
          <property name="text">
           <string>Start Today</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="doneButton">
          <property name="text">
           <string>Done</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionEditor">
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KPIM::KDateEdit</class>
   <extends>QComboBox</extends>
   <header>kdateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
